projects/acryptos/vault/index.js

```diff
@@ -0,0 +1,251 @@
+  "0x93e2e1e384dC298bDEafaEe9751841EA211f2d42", // BTCST-BNB Vault
+  "0x02AaBF12f7C377916BB828BcE3d45c778d919d0e", // SFP-BNB Vault
+  "0x2932568487318969BE7593a27dD5d45aB521e50D", // FRONT-BNB Vault
+  "0xC109d8B9F89Bd939B81Df4Fe47951f9683207102", // WATCH-BNB Vault
+  "0x2D8483Bc2a9E2723711888532Fd542483F041137", // SWINGBY-BNB Vault
+  "0xa1125B756e0ac05ff5d07Cde4D511E1837aADc88", // ALICE-BNB Vault
+  "0x2654346a32D4233B3266AF3C5FD7BAE3C571F345", // FOR-BUSD Vault
+  "0xa387beD33E0415302614545eAD370c27778B955E", // TPT-BUSD Vault
+  "0x373561f3119353e50F21EE1181dd8749ae8276b9", // MIR-UST Vault
+  "0xED51b5c077B71d5B475E30C88B72632fa679fCE3", // COMP-ETH Vault
+  "0x3679d4C2752bEef8632fd12c45b005ecB2774EF0", // SUSHI-ETH Vault
+  "0xe9d9f54Ab89F712ABBdbb3C0F63f2D6eDAa3869c", // BETH-ETH Vault
+];
+
+/*==================================================
+  TVL
+  ==================================================*/
+
+async function tvlInVault(timestamp, block) {
+  const balances = {};
+  const vaultToUnderlyingToken = {};
+
+  // Get acryptos' underlying tokens
+  const underlyingVaultAddressResults = await sdk.bsc.abi.multiCall({
+    calls: _.map(vaultAddresses, (address) => ({
+      target: address,
+    })),
+    abi: vaultAbi["token"],
+  });
+
+  _.each(underlyingVaultAddressResults.output, (token) => {
+    if (token.success) {
+      const underlyingTokenAddress = token.output;
+      const vaultAddress = token.input.target;
+      vaultToUnderlyingToken[vaultAddress] = underlyingTokenAddress;
+      if (!balances[underlyingTokenAddress]) {
+        balances[underlyingTokenAddress] = 0;
+      }
+    }
+  });
+
+  // Get acryptos' balances in underlying token
+  const vaultBalanceResults = await sdk.bsc.abi.multiCall({
+    block,
+    calls: _.map(vaultAddresses, (address) => ({
+      target: address,
+    })),
+    abi: vaultAbi["balance"],
+  });
+
+  _.each(vaultBalanceResults.output, (tokenBalanceResult) => {
+    if (tokenBalanceResult.success) {
+      const valueInToken = tokenBalanceResult.output;
+      const vaultAddress = tokenBalanceResult.input.target;
+      balances[vaultToUnderlyingToken[vaultAddress]] = BigNumber(balances[vaultToUnderlyingToken[vaultAddress]]).plus(
+        valueInToken
+      );
+    }
+  });
+  return balances;
+}
+
+async function tvlInSwapVault(timestamp, block) {
+  const balances = {};
+  let vaultInfos = {};
+  const lpToVault = {};
+
+  // Get vaults' underlying liquidity pairs' addresses
+  const underlyingVaultAddressResults = await sdk.bsc.abi.multiCall({
+    calls: _.map(swapVaultAddresses, (address) => ({
+      target: address,
+    })),
+    abi: vaultAbi["token"],
+    block: block,
+  });
+  vaultInfos = underlyingVaultAddressResults.output.reduce((acc, t) => {
+    if (t.success) {
+      acc[t.input.target] = { lpAddress: t.output };
+      lpToVault[t.output] = t.input.target;
+    }
+    return acc;
+  }, vaultInfos);
+  debug(vaultInfos);
```

Comment:
we can cleanup debug or if we want to leave it we can add a string to describe what's the printed object?